.github/ISSUE_TEMPLATE/offboarding-request.md

```diff
@@ -44,27 +44,33 @@ Please provide the following information about the individual being offboarded:
 
 _The following steps are performed by the **Platform Support** team. Detailed instructions found [here](https://vfs.atlassian.net/wiki/spaces/OT/pages/2097545323/Offboard+Team+Member)_
```

Comment:
Let's update the documentation before merging.

Reply:
@bdech can you remove this line and put the documentation link in the Support Tier 2 devops section similar to what you did for the Tier 1 section below?